core-java-modules/core-java-arrays-convert/src/test/java/com/baeldung/array/conversions/ByteArrayToPrimitiveByteArrayUnitTest.java

```diff
@@ -0,0 +1,38 @@
+package com.baeldung.array.conversions;
+
+import org.apache.commons.lang3.ArrayUtils;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class ByteArrayToPrimitiveByteArrayUnitTest {
+    private static final byte[] EXPECTED_ARRAY_VALUES = {65, 66, 67, 68};
+    private static final Byte[] BYTE_ARRAY = {65, 66, 67, 68};
+
+    @Test
+    void givenByteArray_whenConvertingUsingByteValue_thenGiveExpectedResult() {
+        byte[] newByteArray = new byte[BYTE_ARRAY.length];
+        for (int i = 0; i < BYTE_ARRAY.length; i++) {
+            newByteArray[i] = BYTE_ARRAY[i].byteValue();
+        }
+        Assertions.assertThat(newByteArray)
```

Comment:
you could static import the Assertions:

_import static org.assertj.core.api.Assertions._

Reply:
Fixed